ui/src/lib/components/Link/component.svelte

```diff
@@ -7,12 +7,16 @@
   export let text: string
 </script>
 
-<a
-  {href}
-  {target}
-  on:click|stopPropagation
-  rel="noopener noreferrer"
-  class="font-medium text-blue-600 dark:text-blue-500 hover:underline pr-4"
->
-  {text}
-</a>
+{#if href !== ''}
```

Comment:
kind of a nit: i lean toward the guideline that positives are easy to read than negatives. how do you feel about:

```suggestion
{#if href}
```

Reply:
easy fix, will implement